workflows/resources_test_scripts/annotation_test_data.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# get the root of the directory
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# ensure that the command below is run from the root of the repository
+cd "$REPO_ROOT"
+
+ID=annotation_test_data
+OUT=resources_test/$ID/
+OUT_ONTOLOGY=resources_test/$ID/popv_cl_ontology/
+DIR=$(dirname "$OUT")
+DIR_ONTOLOGY=$(dirname "$OUT_ONTOLOGY")
+
+# ideally, this would be a versioned pipeline run
+[ -d "$DIR" ] || mkdir -p "$DIR"
+[ -d "$DIR_ONTOLOGY" ] || mkdir -p "$DIR_ONTOLOGY"
+
+# dataset page:
+# https://www.10xgenomics.com/resources/datasets/1-k-pbm-cs-from-a-healthy-donor-gene-expression-and-cell-surface-protein-3-standard-3-0-0
+
+
+# Download query h5
+wget https://cf.10xgenomics.com/samples/cell-exp/3.0.0/pbmc_1k_protein_v3/pbmc_1k_protein_v3_filtered_feature_bc_matrix.h5 \
+  -O "${OUT}/tmp_blood_test_query.h5"
+
+
+# # Download Tabula Sapiens Blood reference h5ad
+wget https://www.dropbox.com/s/4cg6zj340oelhlg/Blood.h5ad?dl=1 \
+  -O "${OUT}/tmp_blood_test_reference.h5ad"
+
+# Download PopV specific CL ontology files
+wget https://raw.githubusercontent.com/czbiohub/PopV/main/ontology/cl.obo \
+-O "${OUT_ONTOLOGY}/cl.obo"
+wget https://raw.githubusercontent.com/czbiohub/PopV/main/ontology/cl.ontology \
+-O "${OUT_ONTOLOGY}/cl.ontology"
+wget https://raw.githubusercontent.com/czbiohub/PopV/main/ontology/cl.ontology.nlp.emb \
+-O "${OUT_ONTOLOGY}/cl.ontology.nlp.emb"
+
+
+# Local python MUST have
+#   scanpy==1.8.2
+#   anndata==0.7.8
+python <<HEREDOC
+import scanpy as sc
+import anndata
+assert sc.__version__ == '1.8.2', 'Please make sure to have scanpy 1.8.2 version installed (due to requirement PopV)!'
```

Comment:
I don't see yet why we can't use mudata and anndata>=0.8.
Popv only specifies `scanpy>1.8.0`, no?

→ https://github.com/czbiohub/PopV/blob/main/pyproject.toml#L55
```
scanpy = ">1.8.0"
```